### PR TITLE
Update README for BSIPs 47, 59, 63, 75, 76, 77

### PR DESCRIPTION
**Pull request by ryanRfox**
Updates the README for many new/updated BSIPs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,3 +81,4 @@ Number             | Title                                                    |
 [73](bsip-0073.md) | Match Force-Settlement Orders with Margin Calls and Limit Orders | Abit More | Protocol | Draft
 [74](bsip-0074.md) | Margin Call Fee Ratio | Jerry Liu | Protocol | Draft
 [75](https://github.com/bitshares/bsips/issues/96) | Asset Owner Defines MCR and MSSR Values | John Jones | Protocol | Draft
+[76](https://github.com/bitshares/bsips/issues/221) | ommittee-Defined SmartAsset Collateral Threshold | Abit More | Protocol | Draft
```
Comment (abitmore) [inline README.md]: Typo

Comment (abitmore) [reply README.md]: I'd like to create two BSIPs for the topic. The first one is informational for witnesses to execute via price feeds before the second one is implemented which is a protocol change. So IMHO the title and type listed here doesn't actually reflect the approach.

Comment (ryanRfox) [reply README.md]: Sorry, I neglected to update 76 to Informational when I cut/paste. I'll do that now for 76, then reserve 77 for the Protocol. Reply back here with the titles for both 76 and 77 and I'll update my README doc accordingly. I'll need an Issue # for 77 to point to (assuming you will keep them as distinct discussions).

Comment (abitmore) [reply README.md]: Actually the protocol one only makes sense if the informational one is supported by the majority. I see the potential, but no guarantee. So IMHO don't expect it in the near future, nor need to reserve a number for it so far.

Comment (ryanRfox) [reply README.md]: Good point about the process. I corrected the typo. Still not clear what you prefer the title to be for 76. Separately, assigned 77 for #161

Comment (abitmore) [reply README.md]: Thanks.

**Comment by abitmore** [comment]
Please resolve conflicts.

**Comment by ryanRfox** [comment]
I think that may be correct.

**Pull request merged**